### ISSUE TITLE
论文中比较重要的地方是不是最后的一个BN

### ISSUE THREAD

**Issue by guanyonglai** [automan000/Camera-based-Person-ReID#2]
最后输出特征时加了一个 nn.BatchNorm1d(2048, momentum=None)

**Comment by automan000** [comment]
BN Bottleneck确实在公共数据集上很有价值。详情请见https://arxiv.org/abs/1903.07071
本文中的Baseline也使用了BN Bottleneck。
关于是否使用BN Bottleneck是与网络结构相关的话题，本文重点关注的则是数据输入端的数据分布对齐问题，因此BN Bottleneck并不是本文的重点。
